pkgs/dash_analytics/pubspec.yaml

```diff
@@ -0,0 +1,19 @@
+name: dash_analytics
+description: A package for logging analytics for all dash related tooling to Google Analytics
+version: 0.0.1
+# repository: https://github.com/my_org/my_repo
+
+environment:
+  sdk: '>=2.19.0-406.0.dev <3.0.0'
+
+dependencies:
+  clock: ^1.1.1
+  file: ^6.1.4
+  http: ^0.13.5
+  intl: ^0.17.0
```

Comment:
Can you conform that all these deps are necessary, and aren't trivially removable (inline one method, ...)? I ask since this packages deps will transitively apply to the people who use this package. They're all tools, and may already have all these deps, but if we can easily remove one we should.

Reply:
Yep, all of the dependencies included are needed for required file i/o, working with `DateTime`, and file handling processes. Do we want to pin the versions to a specific version or is this okay?

Reply:
No, these semver constraints look good (no change necessary).